test/services/teams/services/admin.test.js

```diff
@@ -113,6 +113,7 @@ describe("'/teams/manage/admin' service", () => {
 									_id: userId,
 									firstName: 'Hans',
 									lastName: 'Peter',
+									outdatedSince: undefined,
```

Comment:
Can you add a test for the concatenation?

Reply:
the what?

Reply:
A test for the displaying firstname lastname ~~

Reply:
that is not done here